java/org/contikios/cooja/emulatedmote/Radio802154.java

```diff
@@ -55,12 +55,12 @@ public abstract class Radio802154 extends Radio implements CustomDataRadio {
 
     protected boolean isInterfered = false;
 
-    private final boolean isTransmitting = false;
+    private static final boolean isTransmitting = false;
```

Comment:
I think this is a forgotten leftover of an API change. This variable and corresponding get-method should be removed as the sub-classes provide this functionality.